setup.py

```diff
@@ -484,7 +475,7 @@ def get_unique_dependency_list(req_list_iterable: Iterable[list[str]]):
     "celery": celery,
```

Comment:
Separate commit will clean up and move all three celery, kubernetes, dask out of here (and in the docs). They are not needed.